spec/rights_auth_embargo_spec.rb

```diff
@@ -71,7 +71,7 @@
       </objectType>
       EOXML
 
-      expect {Dor::RightsAuth.parse rights}.to raise_error
+      expect {Dor::RightsAuth.parse rights}.to raise_error(ArgumentError, 'argument out of range')
```

Comment:
i am happy to ignore these complaints.  also, i vote for upping line length to 120+.